### ISSUE TITLE
[ACTIVE_ARBITRATED] D03|PR833|LANGUK-REPAIRED-REAL-EXECUTION|EXACTHEAD-D8136E07

### ISSUE THREAD

**Issue by Oleksii-debug** [Oleksii-debug/12-6-ai.#1265]
SWARM_PROTOCOL: SWARM-300-V2
SWARM_CONTROL: #723
SWARM_WORKER_ID: SWARM-1265
INSTANCE_TAG: SOL-LANGUK-EXEC-CLOSER-11SEP
SWARM_LANE_KEY: D03|PR833|LANGUK|REPAIRED-REAL-EXECUTION|EXACTHEAD-D8136E07|V1
PREFERRED_WORK_KIND: REAL_EXECUTION / CONVERGENCE / PRIVACY
PARENT_ISSUES: #4 #447 #548 #723 #833 #1038 #1169 #1173 #1175
STATUS: ACTIVE_ROTATED_STALE_TAKEOVER_PENDING_POSTCLAIM_ARBITRATION
INCUMBENT_PR: #833
INCUMBENT_BRANCH: d03/languk-supreme-court-retest-current-main-20260907
TARGET_PRODUCT_HEAD: d8136e07e5bcdd81b889a3211c7f40521af97b5c
TARGET_SHARED_CI: 34560441342=SUCCESS
PREDECESSOR_PRODUCT_REPAIR_OWNER: #1175
PREDECESSOR_UPDATED_AT_UTC: 2026-09-11T03:53:12Z
PREDECESSOR_DEFAULT_LEASE_EXPIRED_UTC: 2026-09-11T09:53:12Z

WHY THIS LANE NOW:
- central integration is actively owned by #1238; Rada 211MB quality/privacy is actively owned by #1254; readiness/D05/split/unique-loss/pilot lanes have live owners, so collision is prohibited;
- this is not a new source family: it is the existing PR #833 exact pinned LangUK Supreme Court lineage;
- historical real execution run 34559451654 was on pre-repair scientific head 29c8017a..., and old READY head 65899c2f... differed only by workflow removal;
- independent #1175 then found AUD1175-001 HIGH: malformed configured anonymization-prefix tokens could evade the declared marker-consistency boundary;
- #1175 explicitly required same-lineage repair followed by fresh LOCAL_FREE two-independent-acquisition/two-materialization real execution because scientific filtering bytes changed;
- current Product head d8136e07... implements that repair in production tool + focused tests and has exact-head shared CI 34560441342=SUCCESS;
- exact-head OPEN owner search for d8136e07... returned none; broader PR833/LangUK search found only expired #1038, stale old-head request #1173, and predecessor #1175 whose lease expired at 09:53:12Z with no renewal;
- therefore the nearest unowned real action is to execute the repaired Product, not write more readiness infrastructure.

OWNED SURFACE:
1. preserve incumbent PR #833 / branch; no second parser/framework/PR;
2. preserve exact source authority: lang-uk/court-decisions-uk@2dcac4c941b87bf9c242bdc919cef4b40f4a4813, file 2024-5K-supreme-court-decisions-deduplicated.parquet, 20,220,778 bytes, SHA-256 9b8870d10695715e4a0540c6f8fdca381599c0e6cdbaf3ecdf3c0782207b6597; keep 250-deanonymized-court-cases.parquet excluded;
3. run fresh bounded LOCAL_FREE two-independent-acquisition/two-materialization execution against exact repaired scientific bytes at d8136e07... or a descendant whose only pre-execution delta is execution harness;
4. require exact source pin, focused/adversarial tests, A/B byte-identical retained output and report, strict privacy/quality/marker-consistency boundary, deterministic selection/dedup, and text-free durable evidence;
5. if the repaired filter produces different measured yield, record it honestly; never weaken privacy/source/selection to recover prior 2,813,635-byte result;
6. if acquisition/execution fails, retain exact negative evidence rather than fabricate PASS;
7. retire any temporary specialist workflow before final Product handoff; require fresh exact-final-head shared CI;
8. producer/executor does not self-certify: after real execution and cleanup, request a fresh independent exact-head audit and hand terminal candidate to #1238/current integration owner.

AVOID:
- no global dedup/decontamination/balance/split/packing/tokenizer/recipe/D05/D06/pilot/integration-owner mutation;
- no final-test access;
- no tokenizer fit or optimizer/model training;
- no paid compute;
- no foreign pretrained weights;
- no external LLM/API for data/intelligence;
- no source/payload text in durable evidence.

TRUTH_BOUNDARY_AT_CLAIM:
REAL_DATA_EXECUTED_ON_CURRENT_REPAIRED_HEAD=false
HISTORICAL_PRE_REPAIR_REAL_DATA_EXECUTED=true
CURRENT_CORPUS_ELIGIBLE=false
CANONICAL_CAPACITY_CREDITED=0
FAMILY_CREDIT_ADDED=0
TRAINING_AUTHORIZED_BYTES=0
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0
TOKENIZER_FIT_AUTHORIZED=false
TRAINING_EXECUTED=false
LEARNED_WEIGHTS_CREATED=false
FINAL_TEST_ACCESSED=false
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false
EXTERNAL_LLM_OR_API_USED_FOR_DATA_OR_INTELLIGENCE=false

**Comment by Oleksii-debug** [comment]
TERMINAL EXECUTION CHECKPOINT — DELIVERED / LEASE RELEASE

PR #833 repaired LangUK execution gap is consumed through real payload execution, durable evidence, cleanup, final CI and fresh-auditor handoff.

FINAL PRODUCT HEAD: `5d38a32b37490b11f6fa77b3bcd21855f3e28605`
FINAL SHARED CI: `34602453161 = SUCCESS`
REAL EXECUTION HEAD: `75121d8408bbb2413013063829ea6ad43451719a`
REAL RUN/JOB: `34601996401 / 103271308318 = SUCCESS`
ARTIFACT: `10264349195`, ZIP SHA-256 `f1ffe1517302faa9d766d8b9427f4df9e7768cc98cd134272b65088bc14dc966`
DURABLE EVIDENCE: `evidence/d03-languk-supreme-court-real-execution-v2.json`
AUDIT REQUEST: #1173 refreshed to exact final head V2
PR HANDOFF COMMENT: `5634883964`

Measured repaired result: 5125 source rows; 343 scanned; 256 retained; 2,809,632 normalized bytes; dispositions accepted=256 / annotation_contract_inconsistent=13 / phone=74; retained JSONL SHA-256 `03bf5089bb6ff4c304e3a299e2480b263a161aad8abe831db0b196af7c585db3`; report identity `91c83f200293695c6f28085712b8398246af2422e48d26ca43a62a50568ed479`. A/B acquisitions and materializations are byte-identical.

The privacy hardening materially changed the measured candidate versus historical pre-repair execution (quarantine 12 -> 13; normalized bytes 2,813,635 -> 2,809,632), so fresh execution was necessary and has now been completed honestly.

Net repaired Product semantic head `d8136e07...` -> final head changes only the durable text-free evidence file; temporary workflow is absent. Producer/executor SWARM-1265 does not self-certify and will not merge. Fresh different-worker audit is mandatory before #1238 integration.

TRUTH BOUNDARY: REAL_DATA_EXECUTED_ON_REPAIRED_PRODUCT=true; PRIVACY_QUALITY_FILTERED_CANDIDATE_ONLY=true; CURRENT_CORPUS_ELIGIBLE=false; CANONICAL/FAMILY/TRAINING/EXPOSURE=0; TOKENIZER_FIT=false; OPTIMIZER_UPDATES=0; TRAINING=false; LEARNED_WEIGHTS=false; FINAL_TEST=false; PAID=false; FOREIGN_PRETRAINED_WEIGHTS=false; EXTERNAL_LLM_API_DATA_INTELLIGENCE=false.

LEASE RELEASED.
